desktop/dev-environments/index.md

```diff
@@ -3,17 +3,14 @@ description: Dev Environments
 keywords: Dev Environments, share, collaborate, local
 title: Overview
 ---
-
-Dev Environments boosts collaboration by allowing you to share work-in-progress code with your team members. This removes any potential merge conflicts while moving between Git branches to get your code on to their machine.
-
-Dev Environments uses tools built into code editors that allows Docker to access code mounted into a container rather than on your local host. This isolates the tools, files and running services on your machine allowing multiple versions of them to exist side by side.
-
-You can also switch between your developer environments or your team members' environments, move between branches to look at changes that are in progress, without moving off your current Git branch. This makes reviewing PRs as simple as opening a new environment.
-
 > **Beta**
 >
 > The Dev Environments feature is currently in [Beta](../../release-lifecycle.md#beta). We recommend that you do not use this in production environments.
 
+Dev Environments lets you create a configurable developer environment with all the code and tools you need to quickly get up and running. 
+
+It uses tools built into code editors that allows Docker to access code mounted into a container rather than on your local host. This isolates the tools, files and running services on your machine allowing multiple versions of them to exist side by side.
+
 ![Dev environment intro](../images/dev-env.PNG){:width="700px"}
```

Comment:
Just noting that we'll need to update this image with the latest copy. The merged PR from @nicksieger includes a recent screenshot. 

![Dev Envs](https://user-images.githubusercontent.com/154/192042428-54c9c2dc-7826-4fb9-bb0b-be68caf8d185.png)